clearwater-infrastructure/usr/share/clearwater/infrastructure/bin/set_snmp_community

```diff
@@ -29,13 +29,16 @@ then
    exit 1
 fi
 
-if ! [[ $COMMUNITY =~ ^[0-9A-Za-z_-.]*$ ]];
+# Community strings can be any combination of letters, numbers dashes, underscores or dots.
+if ! [[ $COMMUNITY =~ ^[0-9A-Za-z_.\\-]*$ ]];
```

Comment:
Why is there a `\\` here? Wouldn't a single backslash be sufficient?

Reply:
No - slash is a special character so needs to be escaped.